src/cmd_info.rs

```diff
@@ -3,22 +3,27 @@ use crate::{
     wallet::Wallet,
 };
 use prettytable::Table;
+use qr2term::print_qr;
 use std::{error::Error, fs, path::PathBuf, result::Result};
 
-pub fn cmd_info(files: Vec<PathBuf>) -> Result<(), Box<dyn Error>> {
+pub fn cmd_info(files: Vec<PathBuf>, qr_code: bool) -> Result<(), Box<dyn Error>> {
     let mut wallets = Vec::with_capacity(files.len());
     for file in files {
         let mut reader = fs::File::open(&file)?;
         let enc_wallet = Wallet::read(&mut reader)?;
         wallets.push(enc_wallet);
     }
-    print_wallets(wallets);
+    if wallets.len() == 1 && qr_code {
```

Comment:
Does it make sense for it to be an error to pass in more than one wallet and `--qr`?